### DISCUSSION TITLE
怎么配置程序？

### DISCUSSION THREAD

**Discussion by SteveYu000** [Simple-Tracker/qBittorrent-ClientBlocker#58] [General]
看了其他的issue，其中提到有自动屏蔽功能，但我并没有在配置文件中找到这个选项。
另外，对于HP/Torrent/V2.01这个客户端（peer id为HP00001）仍然没能屏蔽，我该如何配置？有相关教程吗？

**Comment by Simple-Tracker** [comment]
1. [配置 Config](https://github.com/Simple-Tracker/qBittorrent-ClientBlocker#%E9%85%8D%E7%BD%AE-config);
2. 更新 3.2 Beta 或手动合并配置文件 config.json;
